smart_control/environment/environment.py

```diff
@@ -35,19 +35,14 @@
 from tf_agents.trajectories import time_step as ts
 from tf_agents.typing import types
 
-from smart_control.models import base_building
-from smart_control.models import base_normalizer
-from smart_control.models import base_reward_function
-from smart_control.proto import smart_control_building_pb2
-from smart_control.proto import smart_control_reward_pb2
-from smart_control.utils import building_image_generator
-from smart_control.utils import constants
-from smart_control.utils import conversion_utils
-from smart_control.utils import histogram_reducer
-from smart_control.utils import plot_utils
-from smart_control.utils import regression_building_utils
-from smart_control.utils import run_command_predictor
-from smart_control.utils import writer_lib
+from smart_control.models import (base_building, base_normalizer,
```

Comment:
For staying consistent with Google import style guidelines, let's use a single line import for each function (previous approach), instead of the parentheses approach. This helps some automated tools better understand what is being imported.